h2o-algos/src/main/java/hex/ensemble/Metalearner.java

```diff
@@ -59,6 +59,12 @@ void computeAutoMetalearner(){
         //GLM Metalearner
         GLM metaGLMBuilder = ModelBuilder.make("GLM", _metalearnerJob, _metalearnerKey);
 
+        //Metalearner parameters
+        if (_hasMetalearnerParams) {
+            GLMModel.GLMParameters glmParams = (GLMModel.GLMParameters) _metalearner_parameters;
+            metaGLMBuilder._parms = glmParams;
+        }
+
```

Comment:
Any reason for the above? How is it different from what is already happening?

Reply:
for some reason, the _metalearner_parameters were completely ignored when computing the autoMetalearner, and I prefer to pass new params through this _metalearner_parameters object rather than adding a new _metalearner_xxx params all the way down.
Actually, maybe we can think about removing those legacy _metalearner_xxx params to rely only on _metalearner_parameters, code would benefit in simplicity and clarity, wdyt?